### ISSUE TITLE
Use different saga type for each action

### ISSUE THREAD

**Issue by gamegee** [anymore1405/redux-toolkit-saga#4]
In the readme example you can pass only one sagaType that apply to the whole slice, but what if I want one action to be called on `takeLatest` and an other one on `normal` (or `takeEvery`)

```tsx
const slice = createSliceSaga({
  name: "testSlice",
  caseSagas: {
    *action1 (action: PayloadAction<string>) {
      yield console.log("ok1", action.payload);
    },
    *action2 (action: PayloadAction<number>) {
      yield console.log("ok2", action.payload);
    },
  },
  sagaType: SagaType.Watch,
});
``` 

One possible API would be something like this:
```tsx
const slice = createSliceSaga({
  name: "testSlice",
  caseSagas: {
    action1: {
      type: SagaType.Watch,
      *fn (action: PayloadAction<string>) {
        yield console.log("ok1", action.payload);
      }
    },
    action2: {
      type: SagaType.TakeLatest,
      *fn (action: PayloadAction<string>) {
        yield console.log("ok2", action.payload);
      }
    }
  }
});
``` 

Btw good work 🔥 
I may create a PR soon to support other SagaTypes

**Comment by anymore1405** [comment]
@gamegee It's a good idea. Thanks for your contribution! 🤗

**Comment by anymore1405** [comment]
This issue has been resolved in new version [1.2.0](https://github.com/anymore1405/redux-toolkit-saga/releases/tag/1.2.0). 
Thanks everyone 🥰
